i18n/locales/az-AZ/blockly-core/core.json

```diff
@@ -10,26 +10,26 @@
     "COLLAPSE_BLOCK": "Bloku yığ",
     "COLOUR_BLEND_COLOUR1": "rəng 1",
     "COLOUR_BLEND_COLOUR2": "rəng 2",
-    "COLOUR_BLEND_HELPURL": "http://meyerweb.com/eric/tools/color-blend/",
+    "COLOUR_BLEND_HELPURL": "",
```

Comment:
Did we lose all these links because of sanitization?

Reply:
Or just so happens these are links but it's because they're not translated they're skipped like the rest? Should these even be in the i18n pipeline since they're only URLs and no string content?

Reply:
Yeah, these are skipped only because they're not translated. It probably doesn't make sense for them to be in the pipeline in the first place, but at least now they're not cluttering up the locales directory!